# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/xfce4/xfcalendar/calendar-icon.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./src/interface.c.orig	2003-09-01 22:50:23.000000000 +0300
+++ ./src/interface.c	2003-12-26 13:52:08.000000000 +0200
@@ -21,7 +21,7 @@
 #include "callbacks.h"
 #include "interface.h"
 #include "support.h"
-#include "xfcalendar-icon-inline.h"
+#include "calendar-icon.h"
 
 #define GLADE_HOOKUP_OBJECT(component,widget,name) \
   g_object_set_data_full (G_OBJECT (component), name, \
@@ -53,7 +53,7 @@
   GtkAccelGroup *accel_group;
   GdkPixbuf *xfcalendar_logo;
 
-  xfcalendar_logo = inline_icon_at_size(xfcalendar_icon, 48, 48);
+  xfcalendar_logo = inline_icon_at_size(calendar_icon_data, 48, 48);
   accel_group = gtk_accel_group_new ();
 
   XFCalendar = gtk_window_new (GTK_WINDOW_TOPLEVEL);
@@ -342,7 +342,7 @@
   GdkPixbuf *xfcalendar_logo;
   gchar *textInfo;
 
-  xfcalendar_logo = inline_icon_at_size(xfcalendar_icon, 48, 48);
+  xfcalendar_logo = inline_icon_at_size(calendar_icon_data, 48, 48);
 
   wInfo = gtk_dialog_new ();
   gtk_widget_set_size_request (wInfo, 300, 370);
